_includes/event.html

```diff
@@ -2,7 +2,7 @@
   <a href="{{ event.url }}">{{event.title}}</a>
   <p> {{ event.content | strip_html | truncatewords: 20 }} </p>
   <div class="meta">
-    {{event.eventdate | date_to_string: "ordinal", "US"  }} 
+    <span class="locale-date">{{ event.eventdate | date_to_rfc822 }}</span>
```

Comment:
We format the date as defined by RFC822 so that Ruby native date is serialized in a predictable way JavaScript can parse.